test/config_test/config_test.cc

```diff
@@ -168,6 +168,8 @@ uint32_t run(const std::string& directory) {
     ENVOY_LOG_MISC(info, "testing {}.\n", filename);
     OptionsImpl options(
         Envoy::Server::createTestOptionsImpl(filename, "", Network::Address::IpVersion::v6));
+    // Avoid contention issues with other tests over the hot restart domain socket.
+    options.setHotRestartDisabled(true);
```

Comment:
Hmm @zuercher is this the same issue you are trying to fix?

Reply:
See https://github.com/envoyproxy/envoy/pull/11016/files. I'm guessing this change can be reverted once the other PR lands.